assets/stylesheets/responsive/_print_style.scss

```diff
@@ -171,6 +172,15 @@ p.subtitle {
     }
 }
 
+.request-header__action-bar {
+    border: 0;
+}
+
+.correspondence__footer__cplink,
```

Comment:
Actually, I've just noticed that core [hides the whole footer](https://github.com/mysociety/alaveteli/blob/develop/app/assets/stylesheets/responsive/_print_style.scss#L41_) – might be worth doing this in a consistent way